src/global.js

```diff
@@ -119,3 +119,22 @@ export function getDisableSetCookie(): boolean {
 
   return false;
 }
+
+export function getExperimentation(): Object {
+  if (typeof __EXPERIMENTATION__ !== "undefined") {
+    if (__EXPERIMENTATION__) {
+      const experimation = {};
+      if (__EXPERIMENTATION__.__EXPERIENCE__) {
+        experimation.experience = __EXPERIMENTATION__.__EXPERIENCE__;
+      }
+      if (__EXPERIMENTATION__.__TREATMENT__) {
+        experimation.treatment = __EXPERIMENTATION__.__TREATMENT__;
+      }
+      return experimation;
+    }
+
+    return null;
+  }
+
+  return null;
```

Comment:
If there are no experiments, then `__EXPERIMENTATION__` should either be `undefined` or look like this:
```
{
    experience: "";
    treatment: "";
}
```

So I think we could adjust this a little bit:
```suggestion
export function getExperimentation(): Experimentation  {
  if (typeof __EXPERIMENTATION__ !== "undefined") {
      return __EXPERIMENTATION__;
  }
  
  return null;
```